### PR TITLE
Implement api to get all rooms

### LINKED ISSUE #291: [API] Get all rooms from a given token

### PR DESCRIPTION
**Pull request by ntchou**
- update swagger doc

### CODE REVIEW

File: functions/src/db/getRooms.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import { Room } from '../types/Room'
+import { db } from '../firebase/firebase'
+
+type Response = Pick<Room, 'id' | 'createdAt' | 'updatedAt'>
+
+export const getRooms = async (userId: UID): Promise<(Response | null)[]> => {
+    const query = await db
+        .collection(COLLECTION_ROOMS)
+        .where('uid', '==', userId)
+    //        .orderBy('createdAt')
```
Comment (HugoGresse) [inline functions/src/db/getRooms.ts:12]: why the comment? :p

**Review by HugoGresse — COMMENTED** [review]
super!

File: functions/src/db/getRooms.ts (hunk 1 above)
Comment (ntchou) [reply functions/src/db/getRooms.ts:12]: It throw an error because i don't created index yet

Comment (ntchou) [reply functions/src/db/getRooms.ts:12]: it's temporary

Comment (HugoGresse) [reply functions/src/db/getRooms.ts:12]: you can update the https://github.com/Instant-Visio/InstantVisio-WebApp/blob/master/firestore.indexes.json so :D

Comment (ntchou) [reply functions/src/db/getRooms.ts]: thx

**Review by HugoGresse — APPROVED** [review]

**Pull request merged**